script.module.t1mlib/addon.xml

```diff
@@ -13,7 +13,7 @@
         <email></email>
         <website></website>
         <forum>http://forum.kodi.tv/showthread.php?tid=247163</forum>
-        <source>https://github.com/learningit/repository.t1m.v2.addons/tree/master/src/script.module.t1mlib</source>
+        <source>https://github.com/xbmc/repo-scripts/tree/isengard/script.module.t1mlib</source>
```

Comment:
I feel your pain :/
Not quite sure how to solve it. We could ask github for an option to disable the download zip button? :D